tdeh/py-tree
# Enhancement/handling for symlinks

## Overview ##
This branch implements handling and displaying of symbolic links.

Resolves #12.

## How to Test ##
* Run unit tests
* Run dir_ascii on a directory that has a symbolic link and confirm that it is printed out properly
